Networking/Networking/Model/Site.swift

```diff
@@ -64,6 +64,10 @@ public struct Site: Decodable, Equatable, GeneratedFakeable, GeneratedCopiable {
     ///
     public let gmtOffset: Double
 
+    /// Whether the site is launched and public.
+    ///
+    public let isBlogPublic: Bool
```

Comment:
nit: "blog" is more for the WordPress blogs, we can drop the blog or refer to "site"
```suggestion
    public let isPublic: Bool
```

Reply:
Renamed in https://github.com/woocommerce/woocommerce-ios/pull/9480/commits/a8b9dd21252ce9faf6cb80f2e528f46e5040e3d8